modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/index.ts

```diff
@@ -323,6 +323,17 @@ function serializeBody(body: BodyInit | null): StructuredClonable {
 	} else {
 		// warn about unserializable body type (eg. FormData, search params,
 		// readable stream),
+		if (body instanceof FormData) {
+			const formData: any = {};
+
+			body.forEach((value: any, key: string) => {
```

Comment:
Here again I would think we want `File | string` instead of `any`.

Reply:
:+1: